crates/bevy_ecs/src/schedule/set.rs

```diff
@@ -112,6 +114,58 @@ impl<T> SystemSet for SystemTypeSet<T> {
     }
 }
 
+/// A [`SystemSet`] implicitly created when using
+/// [`Schedule::add_systems`](super::Schedule::add_systems).
+///
+/// The names of the members are stored in a reference counted slice.
+/// The pointer to the slice is used as hash value and for the equality checks.
+pub struct AnonymousSystemSet {
+    system_names: Arc<[Cow<'static, str>]>,
```

Comment:
I'd like a pub accessor for this field for debug / viz purposes.

Reply:
Done.